Automattic/wp-calypso
# Reader: move teams to redux using middleware

reduxification + middlewarification of ReaderTeams

To Test:
- run `npm run test-client client/state/selectors` and `npm run test-client client/state/reader/teams` and `npm run test-client client/state/data-layer/wpcom/reader`
- make sure calypso build & loads with no errors in the console.  
- Also verify that teams appears in the sidebar

Followup PR: remove old teams code
